addons/mail/static/src/components/channel_member_list/channel_member_list.xml

```diff
@@ -4,21 +4,15 @@
     <t t-name="mail.ChannelMemberList" owl="1">
         <t t-if="channelMemberListView">
             <div class="o_ChannelMemberList d-flex flex-column overflow-auto bg-light" t-attf-class="{{ className }}" t-ref="root">
-                <t t-if="channelMemberListView.onlineCategoryView">
-                    <ChannelMemberListCategory record="channelMemberListView.onlineCategoryView" channelMemberListView="channelMemberListView"/>
+                <t t-if="channelMemberListView.hasMembers">
```

Comment:
Isn't this condition a bit useless / overkill?
There will always be people on a channel (at least the current user in theory) and even in the rare case where no member is known yet, what's the harm in displaying the title with 0?

Reply:
Or maybe it is meant as a temporary wait for when there is 0 because members are not loaded yet? But then it should probably depend on areAllMembersLoaded too: only show nothing if it's 0 and there are missing members, if it's 0 because it's really 0 I feel it should be displayed?

Reply:
I still don't understand why we are making this change at all, and in particular why in the PR just introducing missing models.
Where is even the spec for this change?

Reply:
You are right, I don't think it's useful anymore.